pkg/apply/applydrivers/apply_drivers_default.go

```diff
@@ -67,19 +69,43 @@ type Applier struct {
 
 func (c *Applier) Apply() error {
 	clusterPath := constants.Clusterfile(c.ClusterDesired.Name)
+	c.initStatus()
+	var err error
 	if c.ClusterDesired.CreationTimestamp.IsZero() {
-		if err := c.initCluster(); err != nil {
-			return err
-		}
+		err = c.initCluster()
 		c.ClusterDesired.CreationTimestamp = metav1.Now()
 	} else {
-		if err := c.reconcileCluster(); err != nil {
-			return err
-		}
+		err = c.reconcileCluster()
 	}
+	c.finalStatus(err)
 	logger.Debug("write cluster file to local storage: %s", clusterPath)
 	return yaml.MarshalYamlToFile(clusterPath, c.ClusterDesired)
 }
+func (c *Applier) initStatus() {
+	c.ClusterDesired.Status.Phase = v2.ClusterInProcess
+	c.ClusterDesired.Status.Conditions = make([]v2.ClusterCondition, 0)
+}
+
+func (c *Applier) finalStatus(err error) {
+	logger.Error("final status error: %v", err)
+	condition := v2.ClusterCondition{
+		Type:              "ApplierApplySuccess",
+		Status:            v1.ConditionTrue,
+		LastHeartbeatTime: metav1.Now(),
+		Reason:            "Ready",
+		Message:           "applier apply successfully",
```

Comment:
apply cluster successfully

Reply:
Applied successfully. / Applied to cluster successfully.